src/sources/file.rs

```diff
@@ -83,20 +83,21 @@ pub struct FileConfig {
     /// in `include`. This means that all files are first matched by `include` and then filtered by the `exclude`
     /// patterns. This can be impactful if `include` contains directories with contents that are not accessible.
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "/var/log/binary-file.log"))]
     pub exclude: Vec<PathBuf>,
 
     /// Overrides the name of the log field used to add the file path to each event.
     ///
     /// The value will be the full path to the file where the event was read message.
-    ///
-    /// By default, `file` is used.
     #[serde(default = "default_file_key")]
-    pub file_key: Option<OptionalValuePath>,
+    #[configurable(metadata(docs::examples = "path"))]
+    pub file_key: OptionalValuePath,
 
     /// Whether or not to start reading from the beginning of a new file.
     ///
     /// DEPRECATED: This is a deprecated option -- replaced by `ignore_checkpoints`/`read_from` -- and should be removed.
```

Comment:
If we decide to make all deprecated fields warnings, this will need to be added as a warning in the cue file.

Reply:
No, these have been deprecated and haven't been a part of the documentation for some time - https://v0-15.vector.dev/docs/reference/configuration/sources/file/

Reply:
:phew:

Reply:
Resolved in [192c429](https://github.com/vectordotdev/vector/pull/16099/commits/192c429cac97c0da46a361108a4d292eeef97f2a)